helm/scylla-operator/templates/_helpers.tpl

```diff
@@ -35,7 +35,7 @@ Selector labels
 */}}
 {{- define "scylla-operator.selectorLabels" -}}
 app.kubernetes.io/name: {{ include "scylla-operator.name" . }}
-app.kubernetes.io/instance: {{ .Release.Name }}
+app.kubernetes.io/instance: {{ include "scylla-operator.name" . }}
```

Comment:
These should be hardcoded to `scylla-operator`. Setting nameOverride would break the placement

Reply:
thanks, fixed